codec/cache/build.gradle

```diff
@@ -0,0 +1,13 @@
+plugins {
+    id 'java-library'
+    id 'io.deephaven.project.register'
+}
+
+description 'Codec Cache: Deephaven Codec Cache'
+
+dependencies {
+    implementation project(":codec-base"),
+                   project(":codec-builtin"),
```

Comment:
I was curious why this dependency is necessary. It seems like it's just to support the `io.deephaven.util.codec.CodecCache#getDefaultCodecClass` method. I don't see any callers of that method. I think we can remove it and get rid of the dependency on builtin?

If necessary, we could introduce a ServiceLoader pattern that would allow us to associated types and codecs generically.

Reply:
I think enterprise may use that. We do not.